### ISSUE TITLE
Visibilty (sichtweite) shows wrong unit hPa..

### ISSUE THREAD

**Issue by ThomasWein** [FL550/dwd_weather#118]
## Version of home_assistant
Core: 2024.1.2
Supervisor: 2023.12.0
Operating System: 11.4
Frontend 20240104.0

## Version of the custom_component

v2.1.0
## Describe the bug
HA logged the below mentioned warning for DWD integration. I followed the instruction to fix it in developer statistics but it set it now to the unit **hPa** (my fault 🙈). So I assume that the unit accidently changed during the last update from **km** to **hPa**.

If that's th case can you please fix it with the next update?

## Debug log

```text

2024-01-11 18:55:28.357 WARNING (Recorder) [homeassistant.components.sensor.recorder] The unit of sensor.klippeneck_sichtweite (hPa) cannot be converted to the unit of previously compiled statistics (km). Generation of long term statistics will be suppressed unless the unit changes back to km or a compatible unit. Go to https://my.home-assistant.io/redirect/developer_statistics to fix this
```

**Comment by FL550** [comment]
Yes, this was probably my fault, as some constants within HA have changed and I implemented these changes as well within the last update. I'll release a new version tomorrow which hopefully will fix this.

Thanks for reporting!

**Comment by FL550** [comment]
Fixed in 2.1.1
